src/ray/common/task/task_spec.cc

```diff
@@ -398,11 +398,6 @@ bool TaskSpecification::ExecuteOutOfOrder() const {
          message_->actor_creation_task_spec().execute_out_of_order();
 }
 
-bool TaskSpecification::IsAsyncioActor() const {
-  RAY_CHECK(IsActorCreationTask());
-  return message_->actor_creation_task_spec().is_asyncio();
```

Comment:
We need to delete `is_asyncio` in `ActorCreationTaskSpec` as well.

Reply:
Good catch.